src/internal/connector/exchange/events_restore.go

```diff
@@ -0,0 +1,95 @@
+package exchange
+
+import (
+	"context"
+
+	"github.com/alcionai/clues"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+
+	"github.com/alcionai/corso/src/internal/common/ptr"
+	"github.com/alcionai/corso/src/internal/connector/graph"
+	"github.com/alcionai/corso/src/pkg/backup/details"
+	"github.com/alcionai/corso/src/pkg/fault"
+	"github.com/alcionai/corso/src/pkg/services/m365/api"
+)
+
+var _ itemRestorer = &eventRestoreHandler{}
+
+type eventRestoreHandler struct {
+	ac api.Events
```

Comment:
thought it's a bit opaque, I use `ac` as an initialism for `api.Client`.  I, personally, don't like `api`, as it suggests the declaration of the interface itself, rather than the client which utilizes it.  Likewise, funcs often end up with multiple clients, so I avoid calling any one of them `client`.  With those in mind, `ac` became a simple, unique alternative.  I don't have very strong feelings here, and am open to alternative names (besides `api` and `client`).

Reply:
Fair.  Admittedly, I'm hoping for widespread consistent terminology that allows shorter references in some places.  `ac`, for example, is likely to pop up all throughout the connector package.  But, again, no strong feelings.

Going to dwell on naming and save changes here for a later PR.  We'll be able to modify this with relative ease.